src/routers/controllers/companyAdded.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response, NextFunction } from "express";
+import { CompanyAddSuccess } from "../handlers/yourCompanies/companyAddSuccess";
+import * as constants from "../../constants";
+import logger from "../../lib/Logger";
+
+export const companyAdded = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+        const viewData = await new CompanyAddSuccess().execute(req, res);
+        res.render(constants.COMPANY_ADDED_SUCCESS_PAGE_TEMPLATE, viewData);
+    } catch (e) {
+        return next(e);
```

Comment:
Test coverage indicates that this is not tested

Reply:
I've removed the try { ...} catch(e) next(e) in the async functions and installed express-async-errors instead. We should be able to handle or throw errors in these kind of functions without passing them to next() explicitly.

Still means we will have to deal with those errors in error handler middleware though.